magento/magento2-login-as-customer
# magento/magento2-login-as-customer#150: Opt-in/out. V2

### Description (*)


### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/magento2-login-as-customer#150: Opt-in/out. 
2. Fixes magento/magento2-login-as-customer#152: Admin is able to change "Allow remote shopping assistance" option without access to "Change customer opt-in preference"

### Manual testing scenarios (*)

1. See magento/magento2-login-as-customer#150: Opt-in/out. 
2. See magento/magento2-login-as-customer#152: Admin is able to change "Allow remote shopping assistance" option without access to "Change customer opt-in preference"

### Questions or comments


### Contribution checklist (*)
